python/datasources/cdc_restricted_local.py

```diff
@@ -106,6 +107,7 @@ def accumulate_data(df, groupby_cols, overall_df, demographic_col,
     df[std_col.COVID_HOSP_N] = (df['hosp_yn'] == 'No')
     df[std_col.COVID_HOSP_UNKNOWN] = ((df['hosp_yn'] == 'Unknown') |
                                       (df['hosp_yn'] == 'Missing') |
+                                      (df['hosp_yn'] == 'nul') |
```

Comment:
Is this supposed to be "null"?

Reply:
Offline conversation -- for some reason the data uses "nul" so this is not a typo.